source/meta.tex

```diff
@@ -5334,6 +5461,118 @@
 A reflection of the function designated by \tcode{fn}.
 \end{itemdescr}
 
+%FIXME: inconsistent title case;
+%I'm pretty sure we normally use sentence case.
+%Also, this title is duplicated and weird.
+%Why isn't this called "array reflection" in the style of the subclause above?
+\rSec2[meta.reflection.array]{Promoting to Static Storage}
+
+%FIXME: We generally don't like "compile-time".
+%This sentence is pretty nonsensical because these terms aren't mutually exclusive.
+%
+%We also deal with ranges as input, and ranges aren't necessarily backed by
+%storage (except for an empty class).
+%
+%Maybe we could say something along the lines of converting ranges of structural types
+%into arrays, during program translation.
+\pnum
+The functions in this subclause promote compile-time storage into static storage.
```

Comment:
This is a semi-normative introduction that explains the meaning in simple terms, if you say "converting ranges of structural types into arrays" nobody will know what it does.